src/components/MainMenu/__snapshots__/index.test.js.snap

```diff
@@ -21,7 +21,12 @@ exports[`<MainMenu /> rendering should render Nieuwe Melding NavLink component w
             <span
               className="linklabel"
             >
-              Nieuwe melding
+              <FormattedMessage
```

Comment:
Why isn't the rendered result of the `FormattedMessage` component in the snapshot?

Reply:
Test is using shallow rendering, so child component is being stubbed.

Reply:
But what is the point of having such an output in a snapshot? Wouldn't it be better to have the rendered output in the snapshot?

Reply:
I prefer testing only the file (unit). So that does not include the children.

Also testing all the children is a valid (and even more valuable) test. But mixing these unit tests with more broad tests leads to issues where test break unexpectedly. In addition to being a lot slower.

Reply:
But you're not testing anything here; you're generating a snapshot. If you were to verify that a `FormattedMessage` component is present in the tree, then we would not need a snapshot in the first place.

Reply:
I would have tested the presence of a FormattedMessage. The snapshot was already there, in general I feel these snapshots are used to write less test code rather than do a true snapshot comparison. And there is something to say about ease of use. However I don't want to focus on updating all these old tests in this PR.

Reply:
Fair enough